Makefile

```diff
@@ -0,0 +1,27 @@
+help:
+	@egrep '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-16s\033[0m %s\n", $$1, $$2}'
+
+lint: ## Lint all python code with flake8
+	flake8 --ignore=E501 denbi/ test
+
+test_docker: ## Run tests with docker-compose launch of container
+	docker-compose up -d
+	# Sleep until the container is ready
+	bash -c 'while true; do docker-compose logs --tail=10 | grep "exited: keystone-bootstrap"; ec=$$?; if ((ec==0)); then break; else echo -n .; sleep 1; fi; done;'
+	# Then start testing
+	-python -m unittest test.test_keystone.TestKeystone
+	-python -m unittest test.test_endpoint.TestEndpoint
+	docker-compose kill
+	docker-compose rm -f
+
+test: ## Run tests without docker
+	-python -m unittest test.test_keystone.TestKeystone
+	-python -m unittest test.test_endpoint.TestEndpoint
+
+docs: ## Build documentation
+	rm -f docs/denbi.*.rst
+	sphinx-apidoc -o docs/ denbi/
+	cd docs && $(MAKE) html
+
```

Comment:
`make docs` using python3.5 returns

~~~bash
rm -f docs/denbi.*.rst
sphinx-apidoc -o docs/ denbi/
File docs/denbi.rst already exists, skipping.
Creating file docs/denbi.perun.rst.
Creating file docs/denbi.scripts.rst.
File docs/modules.rst already exists, skipping.
cd docs && make html
make[1]: Entering directory '/home/pbelmann/projects/perunKeystoneAdapter/docs'
Running Sphinx v1.8.1

Exception occurred:
  File "/home/pbelmann/projects/perunKeystoneAdapter/vendor/lib/python3.5/os.py", line 725, in __getitem__
    raise KeyError(key) from None
KeyError: 'DOCUTILSCONFIG'
The full traceback has been saved in /tmp/sphinx-err-om8l27bl.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make[1]: *** [Makefile:20: html] Error 2
make[1]: Leaving directory '/home/pbelmann/projects/perunKeystoneAdapter/docs'
~~~

Reply:
That's very odd, had not seen that before. I'm using 35 as well. it looks like it happened in a newer version of sphinx. I've pinned it in the requirements now to the version I had, 1.7.8